src/richie/apps/search/indexers/courses.py

```diff
@@ -161,20 +224,20 @@ def format_es_object_for_api(es_course, best_language):
         API consumers
         """
         return {
-            "end_date": es_course["_source"]["end_date"],
-            "enrollment_end_date": es_course["_source"]["enrollment_end_date"],
-            "enrollment_start_date": es_course["_source"]["enrollment_start_date"],
             "id": es_course["_id"],
-            "language": es_course["_source"]["language"],
-            "organization_main": es_course["_source"]["organization_main"],
-            "organizations": es_course["_source"]["organizations"],
-            "session_number": es_course["_source"]["session_number"],
-            "short_description": get_best_field_language(
-                es_course["_source"]["short_description"], best_language
+            "start": es_course["_source"]["start"],
```

Comment:
order props alphabetically?

Reply:
in the backend I was trying to put the id first, then dates, then the remaining fields in alphabetical order. WDYT?

Reply:
IMO it's easier to look through props when they are alphabetically ordered unless there's a clear reason to chose another order (or categories like you did with the mappings).

Anyway, it's not a big deal either.